### DISCUSSION TITLE
Consolidate transactions option

### DISCUSSION THREAD

**Discussion by ScoobyDoo24** [davetapley/helium-tax#42] [Ideas]
Tax software like Koinly and Cointracker charge higher rates based on transaction quantity.  Need an option to consolidate all transactions in a specified period (day) to show as a single transaction.  Would be good to also be able to specify week, month, etc.

**Comment by davetapley** [comment]
Thanks for asking, but honestly it's unlikely I'm going to have time to add that functionality.
Moreover, even if I did I'd still be hesitant to expand functionality of this when
you can do filtering and consolidation afterwards.

Perhaps see if you can figure it out using a [pivot table](https://en.wikipedia.org/wiki/Pivot_table) in [one of these](https://en.wikipedia.org/wiki/Pivot_table#Spreadsheets)? 🙏🏻

**Comment by ScoobyDoo24** [comment]
Thanks for taking the time to reply.  Good idea on the pivot table.
